README.md

```diff
@@ -1 +0,0 @@
-# esutil
```

Comment:
Can you put the README back (preferably with more text)? I imagine that needs `TAP_PACKAGE=1` to go in an `eupspkg.cfg.sh` file.